src/dodal/beamlines/i24.py

```diff
@@ -29,6 +30,21 @@
 set_utils_beamline(BL)
 
 
+def attenuator(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> AttenuatorBase:
+    """Get a basic (without filter wheels) attenuator device for i24, instantiate it
```

Comment:
Should: This is a bit misleading, there are filter wheels on the I24 attenuator:
![image](https://github.com/user-attachments/assets/68a22270-1b6d-4b9c-be1b-f67e77e5dd7f).

Reply:
Yes, but for the moment there's no filter wheel device (Made a ticket to write one #927 , which will also be needed soon for i19) and I24 doesn't need it at this stage... Beyond making it clearer in the comment that it's only temporary and will be added in the future I'm not sure what could be added here?